### PR TITLE
Set unified_mode for custom resources

### PR DESCRIPTION
**Pull request by detjensrobert**
- Make wait_for_site checks idempotent
- Set unified_mode for custom resources

### CODE REVIEW

File: test/cookbooks/app_test/libraries/helpers.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module OslAppTest
+  module Cookbook
+    module Helpers
+      def get_return_code(site)
```
Comment (ramereth) [inline test/cookbooks/app_test/libraries/helpers.rb:4]: I'm getting the following similar issue on both app2 and app3 that I'm not sure if it's related to this method or not:
```
       Still waiting for redmine.replicant.us to start... 155 seconds
       
           
           ================================================================================
           Error executing action `run` on resource 'ruby_block[wait_for_replicant]'
           ================================================================================
           
           RuntimeError
           ------------
           Failed to start redmine.replicant.us
```

Comment (detjensrobert) [reply test/cookbooks/app_test/libraries/helpers.rb:4]: A second converge passes, so I suspect the sites are just taking longer to start up. I believe this was happening with the old method as well.

**Review by ramereth — APPROVED** [review]

**Comment by ramereth** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v3.1.0.  Have a nice day!

**Pull request merged**